### ISSUE TITLE
Play button needs Code Runner

### ISSUE THREAD

**Issue by yesenadam** [julia-vscode/docs#28]
Anyone going through the steps in Getting Started will get to "To run the Hello World program, click Run: Julia File in Terminal play button in the top-right side of the editor." and wonder why they don't have a play button like in the picture. After an hour's googling I realized I needed to install the Code Runner extension, not mentioned in these docs. I wasn't sure exactly how I should change the docs to eliminate this problem.

Also just below that currently it says "To learn more about the Julia language, see **Juilalang**.org"

**Comment by flyaflya** [comment]
Yes... the "Hello World" example does not run smoothly for newbies like me.  I had that play button issue and was left wondering where the "Hello" folder came from as well - its creation should be described.

**Comment by StarTrooper08** [comment]
> Anyone going through the steps in Getting Started will get to "To run the Hello World program, click Run: Julia File in Terminal play button in the top-right side of the editor." and wonder why they don't have a play button like in the picture. After an hour's googling I realized I needed to install the Code Runner extension, not mentioned in these docs. I wasn't sure exactly how I should change the docs to eliminate this problem.
> 
> Also just below that currently it says "To learn more about the Julia language, see **Juilalang**.org"

I got your point @yesenadam .
@pfitzseb if you don't mind can I work on this issue??

**Comment by pfitzseb** [comment]
Sure. Take a look at the [Documenter.jl docs](https://juliadocs.github.io/Documenter.jl/stable/) and open a PR.
